### PR TITLE
PAT-536 Change the navigation to the banding banding approval page + styling

### PR DESCRIPTION
**Pull request by yasinmustafa**

### CODE REVIEW

File: server/routes/riskBanding.js [hunk 1]
```diff
@@ -315,6 +315,7 @@ module.exports = function riskBandingConfigRoutes({ offenderService, pathfinderS
           parseInt(nominalId, 10),
           mergedPayload || {},
           userName,
+          user.username,
```
Comment (timharrison-moj) [inline server/routes/riskBanding.js]: Duplicated userName?

File: server/routes/review.test.js [hunk 2]
```diff
@@ -63,6 +63,102 @@ it('returns banding approval page with Banding proposal in the heading', () => {
...
+    .get('/nominal/1/review/banding-assessments')
+    .expect(200)
+    .expect('Content-Type', /html/)
+    .expect(res => {
+      expect(res.text).toContain('approvalButtonSection')
+    })
+})
+
+it('approval button is not displayed for the reviewer', () => {
+  app = appSetup(reviewRoute, '/nominal/:nominalId/review', 'ROLE_PF_APPROVAL')
```
Comment (timharrison-moj) [inline server/routes/review.test.js]: Does this disallow a reviewer from reviewing their own assessments? If so, might cause testing confusion, so perhaps a note to the team to tell them of this feature. I think its right though.

Comment (yasinmustafa) [reply server/routes/review.test.js]: yes, something Eddie wanted.. will inform test team about it.

File: server/routes/riskBanding.js (hunk 1 above)
Comment (yasinmustafa) [reply server/routes/riskBanding.js]: not duplicated username is actually the userId, userName is the firstname + last name

Comment (yasinmustafa) [reply server/routes/riskBanding.js]: not duplicated, userName is the users Firstname + lastname,  user.username is the userId

File: server/routes/review.test.js (hunk 2 above)
Comment (yasinmustafa) [reply server/routes/review.test.js]: Yes it does, will inform the test team about this

**Pull request merged**